DOCS-10463 1.82 (Jun 1) (#209)
mulesoft/docs-api-community-manager: modules/ROOT/pages/diagnostics.adoc

From the Anypoint API Community Manager control panel, open the Salesforce App Launcher and search for *API Community Manager Diagnostics*.

== Status Tab

> The *Status* tab shows the status of your Salesforce *Organization* and your *ACM Communities*.

Reviewer: Throughout, I am just not sure that we need to boldface these when not referring specifically to the label so much as the thing itself: the org, the communities.... Something to review and think about.

Author: Yes, I think using the common nouns is better here. Updated.

. *Published assets status* shows if any assets in API Community Manager incorrectly point to Exchange assets that no longer exist.

If your list of communities is empty, xref:create-community.adoc[Create a Community].

== User Access Tab

> The *User Access* tab shows the status of your *Guest User* and *Member User* configurations, including their permissions.

Reviewer: I was wondering to what "categories" referred when I got to the next para, so maybe here...
```suggestion
The *User Access* tab shows the status of two categories of configuration: *Guest User* and *Member User*, including their permissions.
```

Author: Fixed.

Each category lists its configuration items. Items that are not yet configured correctly have an error and a *Fix* button, and their category has a *Fix All* button. Correctly configured items and categories have no button and a green circle with a white check mark.

*Guest User* items:

* *Profile permission*
* *Classes permission*
* *Sharing rules*

*Member User* items:

* *Community profile*
* *Profile permission*
* *Classes permission*
* *Sharing set*
* *Account created*
* *Account assigned*
* *Account owner role*
* *Account visibility*

> After the items from *Community profile* through *Account created* are correctly configured, assign the Salesforce account manually:

Reviewer: Again, a task in the middle of what is basically a reference topic about the content of a UI element....

. In the control panel, click *Community Administration* > *Login & Registration*.
. Scroll to the section *Registration Page Configuration*.
. Enable *Allow customers and partners to self-register*.
. Set *Registration Page Type* to *Default Page*.

> . Select your community's member user as the *Profile*.

Reviewer: ```suggestion
. Select your community's member user as the value for *Profile*.
```

Author: Fixed.

> . Enter your community's Salesforce account name as the *Account*.

Reviewer: ```suggestion
. Enter your community's Salesforce account name as the value for *Account*.
```

Author: Fixed.

. Click *Save*.

> . In the diagnostics page, next to the *Member User* item *Account assigned*, click the button *Assign* to verify that the account is assigned correctly.

Reviewer: ```suggestion
. In the diagnostics page, next to the *Member User* item *Account assigned*, click *Assign* to verify that the account is assigned correctly.
```

Author: Fixed.
